src/pages/Transaction/TransactionCell/index.tsx

```diff
@@ -269,12 +274,13 @@ const TransactionCellInfo = ({
 }
 
 const TransactionCellCapacityAmount = ({ cell }: { cell: State.Cell }) => {
-  const { udtInfo } = cell
-  return udtInfo && udtInfo.typeHash ? (
+  const { extraInfo } = cell
+  const udtExtraInfo = cell.cellType === 'udt' && extraInfo ? (extraInfo as State.UDTInfo) : null
+  return udtExtraInfo && udtExtraInfo.typeHash ? (
```

Comment:
The type assertion can be avoided by a type guard

Ref: [typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates)